enviro/MeasureCSV/test.csv

```diff
@@ -0,0 +1,3 @@
+HAllo;H;Tst;t
+1;1
+2;2
```

Comment:
This does not look like a nice test file. If it is needed: Please improve it (better variable names, more lines of data) or do not include it in your commit.